src/Model/ProfileField.php

```diff
@@ -46,14 +46,14 @@ class ProfileField extends BaseModel
 	/** @var PermissionSet */
 	private $permissionset;
 
-	/** @var \Friendica\Repository\PermissionSet */
-	private $permissionSetRepository;
+	/** @var \Friendica\Security\PermissionSet\Depository\PermissionSet */
```

Comment:
Please use an import alias, like `PermissionSetDepository`. It should make the constructor declaration lighter on the eyes.